### DISCUSSION TITLE
Run required code mods on version bump

### DISCUSSION THREAD

**Discussion by The-Code-Monkey** [renovatebot/renovate#15034] [Suggest an Idea]
Some packages require codemods or going in and editing it.

Would it be possible to run a codemod against the repo at the same time as bumping the version.

e.g. @types/react@18 has some major changes which requires a codemod.

**Comment by viceice** [comment]
you can use post upgrade tasks on self-hosted renovate.

**Comment by rarkins** [comment]
Where are the codemods hosted, how are they discoverable, etc?
